cli/deepsea.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+"""
+DeepSea CLI
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import logging.config
+import logging
+import os
+import pkg_resources
+import signal
+import sys
+import time
+import click
```

Comment:
This looks like a new dependency; can you add a `Requires: python-click` to the spec file? (We'll also need to make sure we ship this dependency...)

Reply:
It's already there: https://github.com/rjfd/DeepSea/blob/wip-monitor-rebase/deepsea.spec#L57